### PR TITLE
Add an HorizontalScrollView in the keyboard and add virtual keys in it : Home, End, Page Up, Page Down, F1 -> F12

### PR DESCRIPTION
**Pull request by alescdb**
- Add keys in an HorizontalScrollView (except the Keyboard button)
- Create styles for buttons and separators
- Change keyboard autohide timeout from 1500ms to 3000ms to allow better scrolling experience

output

### CODE REVIEW

File: app/src/main/res/layout-large/act_console.xml [hunk 1]
```diff
@@ -142,155 +142,204 @@
...
-				android:contentDescription="@string/image_description_send_tab_character"
-				android:padding="0dp"
-				android:text="@string/button_key_tab"
-				android:textSize="10dip"
-				/>
+				<Button
+						android:id="@+id/button_esc"
+						style="@style/KeyboardButton"
+						android:contentDescription="@string/image_description_send_escape_character"
+						android:text="Esc" />
```
Comment (kruton) [inline app/src/main/res/layout-large/act_console.xml]: Whoops, looks like this didn't come from the @string resources.

File: app/src/main/res/layout-large/act_console.xml [hunk 2]
```diff
@@ -142,155 +142,204 @@
...
+						android:id="@+id/button_tab"
+						style="@style/KeyboardButton"
+						android:contentDescription="@string/image_description_send_tab_character"
 
-		<View
-				android:layout_width="1dp"
-				android:layout_height="match_parent"
-				android:background="#90000000"
-				/>
+						android:text="Tab" />
```
Comment (kruton) [inline app/src/main/res/layout-large/act_console.xml]: This needs to come from `@string/button_key_tab`.

File: app/src/main/res/layout-large/act_console.xml [hunk 3]
```diff
@@ -142,155 +142,204 @@
...
+						style="@style/KeyboardButton"
+						android:text="@string/button_key_pgdn" />
+
+				<View style="@style/KeyboardSeparator" />
+
+				<Button
+						android:id="@+id/button_f1"
+						style="@style/KeyboardButton"
+						android:text="@string/button_key_f1"
+
```
Comment (kruton) [inline app/src/main/res/layout-large/act_console.xml]: nit: extra vertical whitespace.

File: app/src/main/res/layout/act_console.xml [hunk 4]
```diff
@@ -126,155 +126,205 @@
...
-			android:contentDescription="@string/image_description_send_tab_character"
-			android:padding="0dp"
-			android:text="@string/button_key_tab"
-			android:textSize="10dip"
-			/>
+				<Button
+					android:id="@+id/button_esc"
+					style="@style/KeyboardButton"
+					android:contentDescription="@string/image_description_send_escape_character"
+					android:text="Esc" />
```
Comment (kruton) [inline app/src/main/res/layout/act_console.xml]: Needs to come from a `@string` resource.

File: app/src/main/res/layout/act_console.xml [hunk 5]
```diff
@@ -126,155 +126,205 @@
...
+					android:id="@+id/button_tab"
+					style="@style/KeyboardButton"
+					android:contentDescription="@string/image_description_send_tab_character"
 
-		<View
-			android:layout_width="1dp"
-			android:layout_height="match_parent"
-			android:background="#90000000"
-			/>
+					android:text="Tab" />
```
Comment (kruton) [inline app/src/main/res/layout/act_console.xml]: Needs to come from a `@string` resource.

**Comment by alescdb** [comment]
I can't find a correct match for XML codestyle in Android Studio, I've done my changes "by hand" in an external editor.
Maybe you can export your Android Studio codestyle settings and put them into the "config" folder ?

**Comment by jklein24** [comment]
+1 to the codestyle issues. Would be great to get them updated.

**Comment by kruton** [comment]
Just opened #138 with some Android Studio updates and all the XML files reformatted according to those settings.

**Comment by kruton** [comment]
It would be great to have a hint when you first open ConsoleActivity like having this bar visible, then starting like on the second button and scrolling it all the way to the leftmost button. Then that would give a bit of a hint that you can scroll the bar.

**Pull request merged**